internal/cmd/beta/server/describe/describe.go

```diff
@@ -184,35 +187,50 @@ func outputResult(p *print.Printer, model *inputModel, server *iaas.Server) erro
 			table.AddSeparator()
 		}
 
+		if server.ServiceAccountMails != nil && len(*server.ServiceAccountMails) > 0 {
+			table.AddRow("SERVICE ACCOUNTS", strings.Join(*server.ServiceAccountMails, "\n"))
+			table.AddSeparator()
+		}
+
 		if server.Volumes != nil && len(*server.Volumes) > 0 {
 			volumes := []string{}
 			volumes = append(volumes, *server.Volumes...)
 			table.AddRow("VOLUMES", strings.Join(volumes, "\n"))
 			table.AddSeparator()
 		}
 
-		if model.Details {
-			if server.ServiceAccountMails != nil && len(*server.ServiceAccountMails) > 0 {
-				emails := []string{}
-				emails = append(emails, *server.ServiceAccountMails...)
-				table.AddRow("SERVICE ACCOUNTS", strings.Join(emails, "\n"))
-				table.AddSeparator()
-			}
+		err := table.Display(p)
+		if err != nil {
+			return fmt.Errorf("render table: %w", err)
+		}
+
+		// If no --details is set, we only display the overview table
+		if !model.Details {
+			return nil
+		}
+
+		if server.Nics != nil && len(*server.Nics) > 0 {
```

Comment:
why not have just one table with the list of Nics? With a column for each field and a row for each nic

Reply:
Good point, looks much better yes. Since the overview is also a vertical table I had added it the same way